extensions/wrapper/wrapper/src/main/java/de/sovity/edc/ext/wrapper/api/usecase/model/ConsumptionDto.java

```diff
@@ -0,0 +1,18 @@
+package de.sovity.edc.ext.wrapper.api.usecase.model;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@RequiredArgsConstructor
+@Getter
+@Setter
+public class ConsumptionDto {
+    final ConsumptionInputDto input;
+    String contractNegotiationId;
+    String transferProcessId;
```

Comment:
Is this really all the information the user would get? Are these information really that interesting? I Would expect something such as a "state" of the consumption, last change date, etc.